jumpcloud-ADMU/Powershell/Tests/FtaPta.Tests.ps1

```diff
@@ -0,0 +1,111 @@
+# Create test for CSV file creation
```

Comment:
Are we not able to test this in CI?

Reply:
Just uncommented. Yes we can, I commented the whole test file to check on since CI doesn't have associations